mysql-kubernetes/Dockefile

```diff
@@ -0,0 +1,10 @@
+FROM mysql:8.0
```

Comment:
Perhaps better to stick to the 5.x series, as this is what we use everywhere else?